test/instrumentation/testdata/OWNERS

```diff
@@ -0,0 +1,15 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+# To be owned by sig-instrumentation.
+options:
+  no_parent_owners: true
```

Comment:
can you clarify why this is exempted from the normal hierarchy? the reason we've typically done that is for cross-cutting things like API approval, but have tied to an alias rather than specific approvers

Reply:
An alias for this is better. We should probably use `sig-instrumentation-leads`. This directory should be exempted from the normal hierarchy because it functions as a cross-cutting API approval type mechanism for stable metrics. This directory (and the file which will live in it) will be functionally identical to `test/conformance/testdata`.

Reply:
@liggitt Like Han said, goal is to have directory with list of stable metrics that will be reviewed by sig-instrumentation

Reply:
I'm an explicit -1 on `no_parent_owners ` at this time. Both API approvals and conformance testing are under the purview of sig-architecture, who is specifically delegated authority to gate on project-wide design decisions.

I think the regular hierarchy should apply here, with instrumentation being the owners of this directory.

Reply:
@brancz WDYT?

Reply:
Removed it for now to speed up review process.